cleverhans/experimental/certification/dual_formulation.py

```diff
@@ -50,6 +58,9 @@ def __init__(self, sess, dual_var, neural_net_param_object, test_input, true_cla
     self.input_minval = tf.convert_to_tensor(input_minval, dtype=tf.float32)
     self.input_maxval = tf.convert_to_tensor(input_maxval, dtype=tf.float32)
     self.epsilon = tf.convert_to_tensor(epsilon, dtype=tf.float32)
+    self.lzs_params = lzs_params
```

Comment:
shorter syntax (which won't require `if` statement):
`self.lzs_params = lzs_params or DEFAULT_LZS_PARAMS.copy()`

In python `A or B` will return A if A is evaluated to true (i.e. not None and not empty) and B otherwise.